README.md

```diff
@@ -52,7 +54,23 @@ Veneur  (`/vɛnˈʊr/`, rhymes with “assure”) is a distributed, fault-tolera
 
 More generically, Veneur is a convenient sink for various observability primitives with lots of outputs!
 
-See also:
+## Use Case
+
+Once you cross a threshold into O(hundreds) of machines emitting metric data for an application, you've moved into that world where data about individual hosts is uninteresting
```

Comment:
style: `O(unit)` is a bit too colloquial for this format, so I'd replace it with `hundreds` or `hundreds or more`. Though to be honest, I think this statement is true when we're talking even about tens of machines, so maybe `tens or hundreds of machines or more`. Something like that.